matching-banner.diff

```diff
@@ -0,0 +1,75 @@
+diff --git a/frontend/client/components/Proposal/CampaignBlock/index.tsx b/frontend/client/components/Proposal/CampaignBlock/index.tsx
```

Comment:
Was this supposed to get in? I think you may have accidentally checked in the matching banner patch.

Reply:
Sheeeit, good catch.